components/purchase_modal/purchase_modal.tsx

```diff
@@ -233,9 +319,21 @@ class PurchaseModal extends React.PureComponent<Props, State> {
             });
         }
 
+        if (this.props.isDelinquencyModal) {
+            this.props.actions.getInvoices();
+        }
+
         this.props.actions.getClientConfig();
     }
 
+    getDelinquencyTotalString = () => {
+        let totalOwed = 0;
```

Comment:
Optional nit (this already works): What about
```tsx
return `$${this.props.invoices?.reduce((sum, invoice) => sum + invoice.total), 0}/100`
`

Reply:
I wrestled with this for a bit but think it's best to just leave it. Invoices aren't necessarily populated at this point, and causes an error when trying to reduce, which the existing way handles easily